client/my-sites/themes/install-theme-button.jsx

```diff
@@ -65,12 +79,15 @@ const mapStateToProps = ( state ) => {
 		siteSlug: getSelectedSiteSlug( state ),
 		isLoggedIn: isUserLoggedIn( state ),
 		isMultisite: isJetpackSiteMultiSite( state, selectedSiteId ),
+		jetpackSite: isJetpackSite( state, selectedSiteId ),
 		canUploadThemesOrPlugins: siteCanUploadThemesOrPlugins( state, selectedSiteId ),
+		atomicSite: isAtomicSite( state, selectedSiteId ),
 	};
 };
 
 const mapDispatchToProps = ( dispatch ) => ( {
-	dispatchTracksEvent: () => dispatch( recordTracksEvent( 'calypso_click_theme_upload' ) ),
+	dispatchTracksEvent: ( { tracksEventProps } ) =>
+		dispatch( recordTracksEvent( 'calypso_click_theme_upload', tracksEventProps ) ),
```

Comment:
I think this should be `calypso_theme_upload_click` according to convention: PCYsg-4S2-p2

Reply:
Oh, it already exists – ignore me. 🙃